assets/components/svgs/facebook.jsx

```diff
@@ -6,7 +6,14 @@ import React from 'react';
 export default function SvgFacebook() {
 
   return (
-    <svg className="svg-facebook" xmlns="http://www.w3.org/2000/svg" viewBox="0 0 16 16" preserveAspectRatio="xMinYMid">
+    <svg
+      className="svg-facebook"
+      xmlns="http://www.w3.org/2000/svg"
+      viewBox="0 0 16 16"
+      preserveAspectRatio="xMinYMid"
+      aria-labelledby="svg-facebook-title"
+    >
+      <title id="svg-facebook-title">Facebook</title>
```

Comment:
It'd be worth checking how this is read aloud. I assume screen readers would describe it as an image, and then say the word "Facebook"

Reply:
Where these are currently used they're wrapped in a block that has the hint: "Share on social media", so I'm hoping that most screen readers will say something along the lines of "Share on social media... facebook... twitter".

Because these icons are independent of the context in which they are used, I didn't want be too specific with their description, because that might not work if they're used elsewhere. I think it makes sense to allow the place where they are used to give more context.